core/src/main/java/org/elasticsearch/search/aggregations/bucket/range/date/DateRangeAggregationBuilder.java

```diff
@@ -283,7 +288,7 @@ public DateRangeAggregationBuilder addUnboundedFrom(DateTime from) {
     @Override
     protected DateRangeAggregatorFactory innerBuild(SearchContext context, ValuesSourceConfig<Numeric> config,
             AggregatorFactory<?> parent, Builder subFactoriesBuilder) throws IOException {
-        // We need to call processRanges here so they are parsed and we know whether `now` has been used before we make 
+        // We need to call processRanges here so they are parsed and we know whether `now` has been used before we make
         // the decision of whether to cache the request
         Range[] ranges = processRanges(context, config);
```

Comment:
Could we put the check here so that it is run for both the transport and rest APIs? When put here we should make it an `IllegalArgumentException` and we should update the other Range Aggregation Builders too.